### PR TITLE
Office Movers is outdated

### LINKED ISSUE #124: Alias "Movers" to "Moving Company Office"
I typed too many letters (i.e. `mov` works, but `movers` doesn't), and eventually consulted the Wiki to find the appropriate tagging. Can you add a match for `movers?` to point people at `office=moving_company`?

### PR DESCRIPTION
**Pull request by TheAdventurer64**
Fixes #124

### CODE REVIEW

File: data/deprecated.json [hunk 1]
```diff
@@ -1058,6 +1058,9 @@
     "old": {"office": "consultant"},
     "replace": {"office": "consulting"}
   },
+   {
+    "old": {"office": "movers"},
+    "replace": {"office": "moving_company"}
```
Comment (TheAdventurer64) [inline data/deprecated.json:1063]: Forgot to add `},` here.

**Comment by matkoniecz** [comment]
This tag has 2 uses worldwide: https://taginfo.openstreetmap.org/tags/?key=office&value=movers

**Review by matkoniecz — CHANGES_REQUESTED** [review]
It should be handled via alias like in https://github.com/openstreetmap/id-tagging-schema/commit/942e4ce508b02ba3ea5ab516620181b5326e8a6a - not via deprecation

File: data/deprecated.json (hunk 1 above)
Comment (TurnrDev) [reply data/deprecated.json:1063]: A comma on L1063 would in fact be invalid JSON, this is fine.

**Review by TurnrDev — APPROVED** [review]

**Comment by TurnrDev** [comment]
> 
> 
> It should be handled via alias like in [942e4ce](https://github.com/openstreetmap/id-tagging-schema/commit/942e4ce508b02ba3ea5ab516620181b5326e8a6a) - not via deprecation

Actually, yes, you're right. I retract my approval of this change.

**Review by TurnrDev — APPROVED** [review]

**Comment by mbrzakovic** [comment]
Unfortunately, I don't think the recommended change is resolving the issue.
I recommended a more appropriate solution in https://github.com/openstreetmap/id-tagging-schema/issues/124#issuecomment-965094972

**Pull request closed without merging**